src/Pages/SingleArticle.jsx

```diff
@@ -1,13 +1,19 @@
 import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom"
-import { getSingleArticle } from "../utils/api";
+import { getSingleArticle, updateVote } from "../utils/api";
 
 export const SingleArticle = () => {
 
     const [singleArticle, setSingleArticle] = useState([]);
     const [isLoading, setIsLoading] = useState(false)
+    const [count, setCount] = useState(0);
     const { article_id } = useParams();
 
+    const changeVote = () => {
+        updateVote(article_id)
+        setCount(count + 1)
+    }
+
```

Comment:
Just needs some error handling incase it goes wrong. 
At the moment, i can click the button even when offline, and it looks like its going up, but isnt.